integrations/react-ts/src/components/fullFeatureCard.tsx

```diff
@@ -94,11 +109,8 @@ const Index: React.FC = () => {
 
   return (
     <div>
-      <h2>Full Featured Card integration</h2>
       {/* Example below: show loading state */}
       {!elementShow && <p>Loading...</p>}
-      {/* Example below: display response message block */}
-      {errorMessage.length > 0 && <p id="error">{errorMessage}</p>}
```

Comment:
why we move the error show?

Reply:
Drop-In has built-in error alerts. So we don't need to display repetitive error messages.